arwen/host/arwen.go

```diff
@@ -423,6 +433,22 @@ func (host *vmHost) RunSmartContractCall(input *vmcommon.ContractCallInput) (vmO
 	return
 }
 
+func (host *vmHost) createLogEntriesFromErrors(sndAddress, rcvAddress []byte, function string) *vmcommon.LogEntry {
```

Comment:
Only one `LogEntry` is created.
```suggestion
func (host *vmHost) createLogEntryFromErrors(sndAddress, rcvAddress []byte, function string) *vmcommon.LogEntry {
```